youths/schema.py

```diff
@@ -0,0 +1,194 @@
+from datetime import datetime
+
+import graphene
+from django.core import signing
+from django.core.exceptions import ObjectDoesNotExist
+from django_ilmoitin.utils import send_notification
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+
+from profiles.models import Profile
+
+from .consts import GENDERS, LANGUAGES
+from .models import YouthProfile
+from .utils import create_approval_token, mail_encrypted_approval_link
+
+
+class YouthProfileType(DjangoObjectType):
+    class Meta:
+        model = YouthProfile
+        exclude = ["id", "approval_token"]
+
+
+PreferredLanguage = graphene.Enum(
```

Comment:
let's do them in a more human-friendly format, like we did [here](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/schema.py#L41). at least the languages. since labels for gender have apostrophes and spaces, we can use their codes.